deploy/deploy.gradle

```diff
@@ -81,6 +82,29 @@ afterEvaluate { project ->
                         }
                     }
                 }
+
+                pom.withXml {
+                    final dependenciesNode = asNode().appendNode("dependencies")
+                    ext.addDependency = { dep, scope ->
+                        logger.lifecycle("updating dep $dep")
+                        if (dep.group == null || dep.version == null || dep.name == null || dep.name == "unspecified")
+                            return // invalid dependencies should be ignored
```

Comment:
Can we add braces around this?

Reply:
sorry missed the comment, will do this in a follow up